packages/scene-composer/src/components/panels/CustomColorPicker/CustomColorPicker.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useRef, useState } from 'react';
+import { HexColorPicker } from 'react-colorful';
+import { debounce } from 'lodash';
+import { Button, FormField, InputProps, SpaceBetween } from '@awsui/components-react';
+import './CustomColorPicker.scss';
+import { useIntl } from 'react-intl';
+
+import {
+  ColorPickerDivider,
+  ColorPickerWrapper,
+  ColorPreview,
+  FlexibleDiv,
+  RGB,
+  RgbToHex,
+  hexToRgb,
+  isValidHex,
+  validateHex,
+  validateRgbValue,
+  FixedWidthInput,
+} from './CustomColorPickerHelpers';
+
+export const DEFAULT_COLOR_PICKER_COLOR = {
+  hex: '#ffffff',
```

Comment:
This color should come from polaris supported color - https://cloudscape.aws.dev/foundation/visual-foundation/data-vis-colors/. You will have to replace all hard-coded color with polaris supported color.

Reply:
Yes good point, will fix. I used mostly polaris colors but forgot this one🤦🏻‍♂️

Reply:
@TheEvilDev The scene-components/tag-style color picker component is already called ColorPicker, but uses this one. Should we rename that one to TagColorPicker or similar?